platform/android/export/export.cpp

```diff
@@ -2460,10 +2459,10 @@ void register_android_exporter() {
 	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/android/adb", PROPERTY_HINT_GLOBAL_FILE, exe_ext));
 	EDITOR_DEF("export/android/jarsigner", "");
 	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/android/jarsigner", PROPERTY_HINT_GLOBAL_FILE, exe_ext));
-	EDITOR_DEF("export/android/debug_keystore", "");
-	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/android/debug_keystore", PROPERTY_HINT_GLOBAL_FILE, "*.keystore"));
-	EDITOR_DEF("export/android/debug_keystore_user", "androiddebugkey");
-	EDITOR_DEF("export/android/debug_keystore_pass", "android");
+	EDITOR_DEF("export/android/default_keystore", "");
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/android/default_keystore", PROPERTY_HINT_GLOBAL_FILE, "*.keystore"));
+	EDITOR_DEF("export/android/default_keystore_user", "androiddebugkey");
+	EDITOR_DEF("export/android/default_keystore_pass", "android");
```

Comment:
For some reason, when going to Editor Settings after compiling with these changes, I still have the old debug keystore settings showing on top of the new default keystore settings:
<img src="https://user-images.githubusercontent.com/1075032/83386360-3c263380-a3eb-11ea-83c3-56a8b5215c4b.png" width="400" />
Maybe because of some cache?
That could be misleading for users at the moment this change happens, but I have no idea how to fix it.

Reply:
I also saw this happening when testing. It seems there's no "revert" button for unknown settings anymore, unlike what https://github.com/godotengine/godot/issues/22099 says.